packages/itwin/grouping-mapping-widget/src/widget/components/GroupQueryBuilderContainer.tsx

```diff
@@ -10,25 +10,35 @@ import type {
 } from "@itwin/presentation-frontend";
 import { KeySet } from "@itwin/presentation-common";
 import { GroupQueryBuilderApi } from "../../api/GroupQueryBuilderApi";
-import { PropertyGridWrapperApp } from "./property-grid/PropertyGridWrapper";
-import { GroupQueryBuilderContext } from "./context/GroupQueryBuilderContext";
+import { PropertyGridWrapper } from "./property-grid/PropertyGridWrapper";
+import { PropertyGridWrapperContext as PropertyGridWrapperContext } from "./context/PropertyGridWrapperContext";
 import { Button } from "@itwin/itwinui-react";
 import "./GroupQueryBuilder.scss";
+import type { QueryBuilder } from "./QueryBuilder";
+import type { PropertyRecord } from "@itwin/appui-abstract";
 
-export const GroupQueryBuilderContainer: React.FunctionComponent = () => {
+export interface GroupQueryBuilderContainerProps {
```

Comment:
This is a duplicate type. See `GroupingCustomUIProps` (other default query builders use that prop type).

Reply:
Changed